frontend/src/components/MeetingRoom/ChatButton/ChatButton.tsx

```diff
@@ -18,11 +18,13 @@ export type ChatButtonProps = {
  * Also displays an unread message count badge.
  * @param {ChatButtonProps} props - the props for this component
  *   @property {() => void} handleClick - click handler to toggle open chat panel
- *   @property {boolean} isOpen - true if chat is currently open, false if not
+ *   @property {boolean} isOverflowButton - (optional) whether the button is in the ToolbarOverflowMenu
  * @returns {ReactElement} - ChatButton
  */
-const ChatButton = ({ handleClick, isOpen }: ChatButtonProps): ReactElement => {
-  const isSmallViewport = useIsSmallViewport();
+const ChatButton = ({ handleClick, isOverflowButton = false }: ChatButtonProps): ReactElement => {
+  const { rightPanelActiveTab } = useSessionContext();
+  const isOpen = rightPanelActiveTab === 'chat';
```

Comment:
This change was necessary because the `ToolbarButtons` array state change for `rightPanelActiveTab` does not update.